build-support/scripts/up

```diff
@@ -12,12 +12,26 @@ docker run -d --name ad1 --rm \
     -p 48005:48005 \
     -e "NUODB_DOMAIN_ENTRYPOINT=ad1" \
     ${IMG_NAME} nuoadmin
+
+COMMAND="docker exec -i ad1 nuocmd show domain"
+echo -n "Wating for AP to start..."
+while ! $COMMAND 2>/dev/null | grep -q "Connected"; do
+    echo -n "."
+    sleep 2
+done
+echo -e "\nAP started!"
+echo
```

Comment:
You can replace all of this with:

```bash
docker exec ad1 nuocmd check servers --check-leader --check-connected --timeout 60
```

This uses a polling loop, though you might need to guard against the AP not being ready to accept connections at all, since that fails fast.

Just look at `nuocmd check servers -h` output to see the available options. That is definitely better than waiting for `Connected` to appear in the output of `nuocmd show domain`.

See https://doc.nuodb.com/nuodb/latest/reference-information/command-line-tools/nuodb-command/nuocmd-reference/#check-servers

Reply:
Actually, you can disregard my comment about failing fast on connection failures. It looks like the `nuocmd check` commands retry even if the `--api-server` in unreachable, which is what we want:

```console
$ time nuocmd check servers --check-connected --timeout 5
Unable to connect to https://localhost:8888: HTTPSConnectionPool(host='localhost', port=8888): Max retries exceeded with url: /api/1/peers?orderBy=id&limit=20&offset=0 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x10eb79350>: Failed to establish a new connection: [Errno 61] Connection refused',))

real	0m5.311s
user	0m0.178s
sys	0m0.124s
```